packages/core/src/builder.class.ts

```diff
@@ -41,6 +41,7 @@ export const validEnvList = [
   'cloud',
   'fast',
   'cdn2',
+  'forceBuilderCDN',
```

Comment:
```suggestion
  'cdn-prod',
```
Reads more on par with the rest...